README.md

```diff
@@ -84,16 +84,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     League of Women Coders (formerly Ladies Who Code, est. June 2011) is a meetup that focuses on coding, hacking, and learning collaboratively. The group meets up at least once a month in New York City and Boston. The meetups focus on talks, technical questions and ideation, and occasionally feature female speakers.
 
+  - [RodAda Hacker](http://rodadahacker.org/)  - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    The RodAda Hacker is a Brazilian network that provides collaborative workshops of programming specially designed for girls and women. It is a space for those who want to dream, build awesome projects and rethink the technology.
```

Comment:
Rethink the technology? 

That comes off as a bit add, rethink what technology, or rethink technology in general. The rest of it looks quite great.

Reply:
I can put "Rethink about technology" or if you have any suggestions, that will be great!!

Reply:
I think 'Rethink about existing technology' might work for us. That said, if you want, you can drop the section entirely if that's not a big activity there and just marketing text. I'll leave it up to you.

Make a comment here once you are done, and I'll work towards solving the merge conflicts.

Reply:
I'll change. =D 

Sometimes i'm not the great writer so, I guess any tip that i get to make e better text, I embrace it.

Reply:
Hey @hi-hi-ray no worries at all. For a first time contribution, It looks great already 😄